src/chain.h

```diff
@@ -429,11 +429,14 @@ class CChain
 private:
     std::vector<CBlockIndex *> vChain;
```

Comment:
oops , deleted that

Won't compile on native windows
Missing:

#include <atomic>

Reply:
thanks